src/lib/sidebar/PipelineSchemeForm.svelte

```diff
@@ -11,26 +11,20 @@
     TextInput,
   } from "lib/govuk";
   import { gjSchemeCollection } from "stores";
-  import type { SchemeData } from "types";
   import ATF4Type from "../forms/ATF4Type.svelte";
 
   export let scheme_reference: string;
 
   let showModal = false;
-  let scheme: SchemeData | null = null;
-
-  $: scheme = $gjSchemeCollection.schemes[scheme_reference];
-  $: if (scheme) {
-    scheme.pipeline ||= {
-      scheme_type: "",
-      status: "",
-      timescale: "",
-      atf4_lead_type: "",
-      scheme_description: "",
-      funding_source: "",
-      funded: false,
-    };
-  }
+  // This is just for conveience below, but it means most changes are not
```

Comment:
It's quite frustrating that the Svelte compiler can't trace through a simple alias for typing convenience like this. Maybe something in Svelte 5 / runes improves it.